api/src/db/foreign/base.py

```diff
@@ -0,0 +1,41 @@
+#
+# SQLAlchemy base model and metadata for foreign tables.
+#
+# These are defined in a separate sqlalchemy.MetaData object as they are managed differently:
+#  - The tables are actually foreign tables connected to the source Oracle database in most environments.
+#  - System components should not access them, except transform related components.
+#  - Migrations are not used to manage creation and changes as the tables are actually defined in a different system.
+#
+
+import datetime
+from typing import Any, Iterable
+
+import sqlalchemy
+
+from src.constants.schema import Schemas
+
+metadata = sqlalchemy.MetaData()
+
+
+class Base(sqlalchemy.orm.DeclarativeBase):
```

Comment:
Could we give this (and the file) a different name? Otherwise it can be easy to make a mistake when importing the right "Base" class.